packages/cli/src/checks/__tests__/EXTERNAL_MISMATCH.ts

```diff
@@ -263,32 +263,32 @@ it("should not error if the range is included in the allowedDependencyVersions o
 it("should error and fix the version to the closest allowed one when adding an allowed major", () => {
   const options = {
     allowedDependencyVersions: {
-      something: ["1.0.0", "2.0.0"]
+      something: ["^1.0.0", "^2.0.0"]
```

Comment:
Prob worth retaining a mixture of with and without carets in the tests

Reply:
Yep, will do a mixture here as well